playbooks/email_notification_for_malware.yml

```diff
@@ -0,0 +1,22 @@
+name: Email Notification for Malware
+id: fb3edc76-ff2b-48b0-5f6f-63da6483fd63
+version: 1
+date: '2021-01-19'
+author: Philip Royer, Splunk
+type: Response
+description: This playbook tries to determine if a file is malware and whether or not the file is present on any managed machines. VirusTotal "file reputation" and PAN WildFire "detonate file" are used to determine if a file is malware, and CarbonBlack Response "hunt file" is used to search managed machines for the file. The results of these investigations are summarized in an email to the incident response team.
+playbook: email_notification_for_malware
+how_to_implement: "Be sure to update asset naming to reflect the asset names configured in your environment."
+references: []
+app_list:
+- "VirusTotal"
+- "WildFire"
+- "CarbonBlack Response"
+- "SMTP"
+tags:
+  platform_tags:
+  - Response
+  playbook_fields:
+  - FileHash
```

Comment:
Should be fileHash because SOAR is case-sensitive

Reply:
Should also have the vaultId field which is used for "detonate file"

Reply:
Fixed